git-fs.c

```diff
@@ -27,6 +29,20 @@ bool enable_debug = 0;
 
 #define error(...) fprintf(stderr, __VA_ARGS__)
 
+// Dump a stacktrace to stderr
+static void dump_trace(int signum) {
+	error("Segmentation fault\n");
+	void * buffer[255];
+	const int calls = backtrace(buffer, sizeof(buffer) / sizeof(void *));
```

Comment:
Yes, I'm not saying that you're supposed to pass the size in bytes, I'm saying that I'm already passsing the number of adresses that can be stored. The buffer is `sizeof(buffer)` big, a single address is `sizeof(void*)` big, so dividing those two gives you the number of addresses that fit in `buffer`?